scripts/jenkins/Jenkinsfile

```diff
@@ -859,22 +889,20 @@ def buildPackageDebianStretch() {
   }]
 }
 
-def deployHomepage() {
```

Comment:
why the strange notation of having to deploy 2 images?

just split it up to 1 image

Reply:
this could probably be further improved:

* Instead of passing the imageId I think it is easier to just pass the image description.
* host should probably be renamed to hostname.